javadoc error fails build with distribution profile on JDK8

```bash
$ java -version
java version "1.8.0_51"
Java(TM) SE Runtime Environment (build 1.8.0_51-b16)
Java HotSpot(TM) 64-Bit Server VM (build 25.51-b03, mixed mode)

$ mvn -Pdistribution package
...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] ADAM_2.10 .......................................... SUCCESS [  7.118 s]
[INFO] ADAM_2.10: Core .................................... FAILURE [  8.876 s]
[INFO] ADAM_2.10: APIs for Java ........................... SKIPPED
[INFO] ADAM_2.10: CLI ..................................... SKIPPED
[INFO] ADAM_2.10: Distribution ............................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 16.235 s
[INFO] Finished at: 2015-08-26T17:20:24-05:00
[INFO] Final Memory: 36M/640M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.9.1:jar (attach-javadocs) on project adam-core_2.10: MavenReportException: Error while creating archive:
[ERROR] Exit code: 1 - /Users/xxx/working/adam/adam-core/src/main/java/org/bdgenomics/adam/io/InterleavedFastqInputFormat.java:55: error: unknown tag: date
[ERROR] * @date Feb 2014
[ERROR] ^
[ERROR] /Users/xxx/working/adam/adam-core/src/main/java/org/bdgenomics/adam/io/InterleavedFastqInputFormat.java:206: warning: no @return
[ERROR] public Text createKey() {
[ERROR] ^
[ERROR] /Users/xxx/working/adam/adam-core/src/main/java/org/bdgenomics/adam/io/InterleavedFastqInputFormat.java:213: warning: no @return
[ERROR] public Text createValue() {
[ERROR] ^
[ERROR] /Users/xxx/working/adam/adam-core/src/main/java/org/bdgenomics/adam/io/InterleavedFastqInputFormat.java:220: warning: no @return
[ERROR] public long getPos() {
[ERROR] ^
[ERROR] /Users/xxx/working/adam/adam-core/src/main/java/org/bdgenomics/adam/io/InterleavedFastqInputFormat.java:266: warning: no @param for value
[ERROR] public boolean next(Text value) throws IOException {
[ERROR] ^
[ERROR] /Users/xxx/working/adam/adam-core/src/main/java/org/bdgenomics/adam/io/InterleavedFastqInputFormat.java:266: warning: no @return
[ERROR] public boolean next(Text value) throws IOException {
[ERROR] ^
[ERROR] /Users/xxx/working/adam/adam-core/src/main/java/org/bdgenomics/adam/io/InterleavedFastqInputFormat.java:266: warning: no @throws for java.io.IOException
[ERROR] public boolean next(Text value) throws IOException {
[ERROR] ^
[ERROR] /Users/xxx/working/adam/adam-core/src/main/java/org/bdgenomics/adam/io/SingleFastqInputFormat.java:47: error: unknown tag: date
[ERROR] * @date September 2014
[ERROR] ^
[ERROR] /Users/xxx/working/adam/adam-core/src/main/java/org/bdgenomics/adam/io/SingleFastqInputFormat.java:196: warning: no @return
[ERROR] public Text createKey() {
[ERROR] ^
[ERROR] /Users/xxx/working/adam/adam-core/src/main/java/org/bdgenomics/adam/io/SingleFastqInputFormat.java:203: warning: no @return
[ERROR] public Text createValue() {
[ERROR] ^
[ERROR] /Users/xxx/working/adam/adam-core/src/main/java/org/bdgenomics/adam/io/SingleFastqInputFormat.java:210: warning: no @return
[ERROR] public long getPos() {
[ERROR] ^
[ERROR] /Users/xxx/working/adam/adam-core/src/main/java/org/bdgenomics/adam/io/SingleFastqInputFormat.java:255: warning: no @param for value
[ERROR] public boolean next(Text value) throws IOException {
[ERROR] ^
[ERROR] /Users/xxx/working/adam/adam-core/src/main/java/org/bdgenomics/adam/io/SingleFastqInputFormat.java:255: warning: no @return
[ERROR] public boolean next(Text value) throws IOException {
[ERROR] ^
[ERROR] /Users/xxx/working/adam/adam-core/src/main/java/org/bdgenomics/adam/io/SingleFastqInputFormat.java:255: warning: no @throws for java.io.IOException
[ERROR] public boolean next(Text value) throws IOException {
[ERROR] ^
[ERROR] 
[ERROR] Command line was: /Library/Java/JavaVirtualMachines/jdk1.8.0_51.jdk/Contents/Home/bin/javadoc @options @packages
```

--- a/adam-core/src/main/java/org/bdgenomics/adam/io/InterleavedFastqInputFormat.java
+++ b/adam-core/src/main/java/org/bdgenomics/adam/io/InterleavedFastqInputFormat.java
@@ -52,7 +52,6 @@ import java.io.InputStream;
  * found at https://github.com/HadoopGenomics/Hadoop-BAM/blob/master/src/main/java/org/seqdoop/hadoop_bam/FastqInputFormat.java
  *
  * @author Jeremy Elson (jelson@microsoft.com)
- * @date Feb 2014
  */
 public class InterleavedFastqInputFormat extends FileInputFormat<Void,Text> {
     

--- a/adam-core/src/main/java/org/bdgenomics/adam/io/SingleFastqInputFormat.java
+++ b/adam-core/src/main/java/org/bdgenomics/adam/io/SingleFastqInputFormat.java
@@ -44,7 +44,6 @@ import java.io.InputStream;
  * found at https://github.com/HadoopGenomics/Hadoop-BAM/blob/master/src/main/java/org/seqdoop/hadoop_bam/FastqInputFormat.java
  *
  * @author Frank Austin Nothaft (fnothaft@berkeley.edu)
- * @date September 2014
  */
 public class SingleFastqInputFormat extends FileInputFormat<Void,Text> {
     

